Foundation/NSURLSession/NSURLSessionTask.swift

```diff
@@ -630,9 +630,12 @@ fileprivate extension URLSessionTask {
         var result = [("Connection", "keep-alive"),
                       ("User-Agent", userAgentString),
                       ]
+        #if !os(Android)
+        // Crashes on Android https://bugs.swift.org/browse/SR-2588
```

Comment:
The bug claims no stack trace, without one the chances of this being fixed is low. Does lldb not work for Android? it should be able to interface with the gdbserver loaded with the apk.